galaxy.yml

```diff
@@ -3,7 +3,7 @@
 # The namespace of the collection. This can be a company/brand/organization or product namespace under which all
 # content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
 # underscores or numbers and cannot contain consecutive underscores
-namespace: maxamillion
+namespace: consoledot
 
 # The name of the collection. Has the same character restrictions as 'namespace'
 name: fleetmanager
```

Comment:
Does this need to be changed to edgemanagement?

Reply:
yes it does, thanks!